apps/wallet/src/ui/styles/global.scss

```diff
@@ -13,6 +13,16 @@
     }
 }
 
+@layer components {
+    .custom-scrollbar::-webkit-scrollbar {
```

Comment:
there's an un-official plugin called `tailwind-scrollbar`, but the classes auto-apply `overflow: overlay` which causes problems with how we want to layout the list and scrollbar. I just ditched it and went along with the simplest solution

Reply:
Yeah, `overflow: overlay` is also deprecated. We probably want a good long-term solution here that gives us the UX we want, like: https://kingsora.github.io/OverlayScrollbars/ (we've talked about styling scrollbars before)

Can you open a JIRA just to track this/

Reply:
Agreed! https://mysten.atlassian.net/browse/APPS-604